### ISSUE TITLE
[BUG] Wrong precision for TRX/BTC pair on Binance

### ISSUE THREAD

**Issue by mifeng** [CoinAlpha/hummingbot#1618]
**Describe the bug**
Hummingbot uses 7 precision for the TRX/BTC pair on Binance, but Binance API and order book uses 8

**Screenshots**

TRX/BTC in Hummingbot:
![Question](https://user-images.githubusercontent.com/1795030/79173192-ac311980-7dab-11ea-857e-4da165f3086d.JPG)

TRX/BTC order book:
![Screen_Shot_2020-04-13_at_5 20 12_PM](https://user-images.githubusercontent.com/1795030/79173205-b7844500-7dab-11ea-9de3-8b9a31d11754.png)

Binance API `https://api.binance.com//api/v1/exchangeInfo`:
```
symbol: "TRXBTC",
status: "TRADING",
baseAsset: "TRX",
baseAssetPrecision: 8,
quoteAsset: "BTC",
quotePrecision: 8,
baseCommissionPrecision: 8,
quoteCommissionPrecision: 8,
```

**Comment by dennisocana** [comment]
While Clinton and I were testing this today we found out that this bug also causes problems with sell orders not being created for BTC pairs with prices of less than 1,000 Satoshi. 

https://docs.google.com/spreadsheets/d/1_-osX5H2kHAPtBWXF6vAzRx3zntsvhkrI6Hcwgod9wY/edit#gid=0

Confirmed fix in the PR created.

![cmd_9NjfxBkYAJ](https://user-images.githubusercontent.com/50150287/79322560-95bbc700-7f3f-11ea-8775-ad13fa7bae91.png)

**Comment by dennisocana** [comment]
Fix will be in the upcoming `0.26.0` release.
